packages/wrangler/src/pages/dev.ts

```diff
@@ -252,23 +259,33 @@ export const Handler = async ({
 	let scriptPath = "";
 
 	if (usingWorkerScript) {
-		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-		scriptReadyResolve!();
-
 		scriptPath = workerScriptPath;
-
-		const runBuild = async () => {
-			try {
-				await esbuild.build({
-					entryPoints: [scriptPath],
-					write: false,
-					// we need it to be bundled so that any imports that are used are affected by the blocker plugin
-					bundle: true,
-					plugins: [blockWorkerJsImports],
-				});
-			} catch {}
+		let runBuild = async () => {
+			await checkRawWorker(workerScriptPath, () => scriptReadyResolve());
```

Comment:
Codecov is mentioning these lines are uncovered by tests? Is this covered in the fixtures rather than the wrangler package tests (Codecov doesnt collect fixtures tests I think)

Reply:
Yeah this is covered by fixtures tests.